src/main/scala/mesosphere/marathon/core/plugin/impl/PluginManagerImpl.scala

```diff
@@ -18,7 +23,7 @@ import scala.reflect.ClassTag
   * The plugin manager can load plugins from given urls.
   * @param urls the urls pointing to plugins.
   */
-private[plugin] class PluginManagerImpl(val descriptor: PluginsDescriptor, val urls: Seq[URL]) extends PluginManager {
+class PluginManagerImpl(val descriptor: PluginsDescriptor, val urls: Seq[URL]) extends PluginManager {
```

Comment:
Why did you change the visibility of this class? It doesn't seem like it is being used outside of the plugin package.